component/sections/call_to_action/variant_d.js

```diff
@@ -37,17 +37,19 @@ function VariantD({
             {button && (
               <a
                 className="inline-block py-2 px-6 bg-webriq-blue hover:bg-webriq-darkblue text-white font-bold leading-loose transition duration-250 rounded-l-xl rounded-t-xl"
-                target={button?.pageAccess === "openLinkToNewTab" ? "_blank" : null}
-                rel={button?.pageAccess === "openLinkToNewTab" ? "noopener noreferrer" : null}
+                target={button?.linkTarget}
+                rel={
+                  button?.linkTarget === "_blank" ? "noopener noreferrer" : ""
```

Comment:
When condition returns false, it will set value to empty string vs null (nothing).